tests/test_views.py

```diff
@@ -365,27 +366,27 @@ def test_env_reverse_proxy_path(proxy_path, client):
 @pytest.mark.slow
 def test_env_theme(theme, client):
     dom = assert_response(client.get('/'), '/')
-    assert dom.xpath('//head/link[@rel="stylesheet"][@href=$href]',
-                     href=f'/static/admin/bootstrap/bootstrap3/swatch/{theme or "default"}/bootstrap.min.css?v=3.3.5')
+    assert dom.xpath('//head/link[@rel="stylesheet"][starts-with(@href, $href)]',
+                     href=f'/static/admin/bootstrap/bootstrap3/swatch/{theme or "default"}/bootstrap.min.css?')
```

Comment:
We don't actually care about _precise_ version numbers here; we're only checking the theme name